### PR TITLE
fix: replace og:image

### PR DESCRIPTION
**Pull request by caugner**
### Motivation

We get 40k weekly hits on `https://developer.mozilla.org/static/img/opengraph-logo.72382e605ce3.png`, which 404s, because `opengraph-logo.png` has been replaced by [mdn-social-share.png](https://developer.mozilla.org/mdn-social-share.png).

### Related issues and pull requests

See also https://github.com/mdn/content/pull/27046 for the same change in content.

### CODE REVIEW

File: files/zh-cn/web/api/performance/clearresourcetimings/index.md
Comment (jasonren0403) [inline files/zh-cn/web/api/performance/clearresourcetimings/index.md:1]: The code example in this file may be updated afterwards(full content also found outdated). Except for this, no other problems for l10n-zh 👍

**Review by jasonren0403 — APPROVED** [review]

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es`

Also taking a note about @mfuji09 says

**Review by josielrocha — APPROVED** [review]
LGTM for @mdn/yari-content-pt-br!

**Review by cw118 — APPROVED** [review]
LGTM for l10n-fr, thanks very much!

**Comment by caugner** [comment]
@mfuji09 Thanks again for your comment regarding absolute URLs. 🙌  This is now fixed in [3270cd0](https://github.com/mdn/translated-content/pull/13501/commits/3270cd0c3f447448fba4694f418db765fa423386).

**Pull request merged**